elastic/kibana
# [Response Ops][Actions] Adding configuration to override default MS Graph API Scope and Exchange URL values

Resolves https://github.com/elastic/kibana/issues/166064

## Summary

Adds the following configurations to the `kibana.yml` config:
* `xpack.actions.microsoftGraphApiScope` - overrides the default Graph API scope value of `https://graph.microsoft.com/.default`
* `xpack.actions.microsoftExchangeUrl` - overrides the default value of `https://login.microsoftonline.com`

This allows users in different Azure environments to customize their endpoints as needed.

## To Verify

We are unable to test this in a different environment but we can verify that the config overrides the defaults as expected by setting the config values to something different and the logging out the params that are sent to `getOAuthClientCredentialsAccessToken` in `x-pack/plugins/stack_connectors/server/connector_types/email/send_email.ts`. Then create an MS Exchange email connector and test it to see that the logged values are overridden as expected.
